docs/not-alma.md

```diff
@@ -17,6 +17,8 @@
 |----------|:-----------:|:------:|:-----------:|:----------------:|
 | <span style="display: inline-block; vertical-align: middle;"><img src="docs/images/anytype.png" alt="Anytype" style="width: 30px; height: 30px; border-radius: 10px;"> </span> <span style="display: inline-block; vertical-align: middle;"> Anytype </span> | <span style="color: green;">✓</span> | Özel Lisans | Lokal | <i class="fa-brands fa-windows"></i> <i class="fa-brands fa-apple"></i> <i class="fa-brands fa-linux"></i> <i class="fa-brands fa-android"></i> <i class="fa-brands fa-app-store-ios"></i>  |
 | <span style="display: inline-block; vertical-align: middle;"><img src="docs/images/logseq-icon.png" alt="logseq" style="width: 30px; height: 30px;"> </span> <span style="display: inline-block; vertical-align: middle;"> Logseq </span> | <span style="color: green;">✓</span> | AGPL-3.0 | Lokal | <i class="fa-brands fa-windows"></i> <i class="fa-brands fa-apple"></i> <i class="fa-brands fa-linux"></i> <i class="fa-brands fa-android"></i> <i class="fa-brands fa-app-store-ios"></i> |
+| <span style="display: inline-block; vertical-align: middle;"><img src="docs/images/notesnook-icon.png" alt="notesnook" style="width: 30px; height: 30px;"> </span> <span style="display: inline-block; vertical-align: middle;"> Notesnook </span> | <span style="color: green;">✓</span> | AGPL-3.0 | Lokal-Bulut | <i class="fa-solid fa-globe"></i> <i class="fa-brands fa-windows"></i> <i class="fa-brands fa-apple"></i> <i class="fa-brands fa-linux"></i> <i class="fa-brands fa-android"></i> <i class="fa-brands fa-app-store-ios"></i> |
+| <span style="display: inline-block; vertical-align: middle;"><img src="docs/images/joplin-icon.png" alt="joplin" style="width: 30px; height: 30px;"> </span> <span style="display: inline-block; vertical-align: middle;"> Joplin </span> | <span style="color: green;">✓</span> | MIT| Lokal | <i class="fa-brands fa-windows"></i> <i class="fa-brands fa-apple"></i> <i class="fa-brands fa-linux"></i> <i class="fa-brands fa-android"></i> <i class="fa-brands fa-app-store-ios"></i> |
```

Comment:
Joplin de anladığım kadarıyla MIT değil [AGPL lisansına](https://github.com/laurent22/joplin?tab=License-1-ov-file) sahip.

Orjinal repo (https://github.com/joplin) indirme olarak başka bir repoya, maintainer'in sahsi reposuna, (https://github.com/laurent22/joplin) yönlendiriyor.

Reply:
Ayrıca ücretli olup olmadığından emin olmamakla birlikte cloud versiyonu da mevcut imiş.